packages/store/src/internal/state-factory.ts

```diff
@@ -60,7 +60,7 @@ import { ensureStateNameIsUnique, ensureStatesAreDecorated } from '../utils/stor
 import { ensureStateClassIsInjectable } from '../ivy/ivy-enabled-in-dev-mode';
 import { NgxsUnhandledActionsLogger } from '../dev-features/ngxs-unhandled-actions-logger';
 
-const NG_DEV_MODE = typeof ngDevMode === 'undefined' || ngDevMode;
+const NG_DEV_MODE = typeof ngDevMode !== 'undefined' && ngDevMode;
```

Comment:
There are many variants of this throughout the codebase.
Which is correct?
![image](https://github.com/ngxs/store/assets/1948265/0b673f21-6c5a-479e-ae99-015d7e1fa96e)

Reply:
I forgot to revert.